### DISCUSSION TITLE
KICKOFF 미팅

### DISCUSSION THREAD

**Discussion by bryantson** [Korean-American-IT-Association-KAITA/kaita-project-management#18] [2. 자유토크]
첫 킥오프 미팅 가능한 시간 적어 주세요!

예: 월-금: 9PM CST 주말: 8PM

**Comment by jparksecurity** [comment]
월-목 7PM-9PM CST 주말 아무떄나

**Comment by yenny1124** [comment]
금, 토, 일 아무때나

**Comment by bbkaeul** [comment]
화 - 토 6:00CST -24:00 CST 아무때나

**Comment by j2jun** [comment]
금, 토 - 7pm-9pm cst

**Comment by ndk6879** [comment]
가능한 시간: 토 CST 7pm ~ 9pm

**Comment by ghkfehd0502** [comment]
5pm CST 이후

**Comment by judyyhham** [comment]
월, 화, 금, 토 6 pm CST~

**Comment by klee5760** [comment]
흠...
밤 9시부터 12시까지 CST 가능합니다.

**Comment by jung-akim** [comment]
월, 화, 수, 목, 일 7pm - 9pm CST

**Comment by ghost** [comment]
월-목 9-11pm CST / 일 8시 이후

**Comment by yeinsong** [comment]
평일
16:00 ~ 19:00 CST
06:00 ~ 12:00 CST

주말 (금, 토, 일)
19:00 ~ CST

**Comment by kikibell1004** [comment]
주말 금토일 저녁시간 가능합니다.

**Comment by tosky1125** [comment]
Kick off 미팅 인비는 따로 전달해주시나요?!

**Reply by bryantson** [reply]
곧 보내겠습니다

**Comment by kikibell1004** [comment]
주말 아무때나 가능합니다~ 아님 금요일 밤

**Comment by bryantson** [comment]
### 스케줄 계획표 만들었습니다. 직접 바꿀수 있으니 한번 확인해주시고 고쳐야 하면 고쳐 주세요.

https://docs.google.com/spreadsheets/d/1_7XDQq-XCNYgHgqrp4c0zmFLwOm3JGT9CyZCf8iyJF8/edit?usp=sharing

첫 킥오프 미팅은 10월 28일 (미국 - 목요일) 이나 그 이후 생각 중입니다

cc: @jparksecurity @yenny1124 @bbkaeul @josephjun8 @ndk6879 @ghkfehd0502 @judyyhham @klee5760 @jung-akim @jungwon-oh87 @yeinsong @kikibell1004 @tosky1125

**Reply by jparksecurity** [reply]
https://doodle.com/poll/w3xhhbt9vskvqcrc?utm_source=poll&utm_medium=link

Doodle 한번 만들어 보았습니다. 변경해야할 부분 있으면 말씀해주세요

**Reply by bryantson** [reply]
오 두들 좋네요 ^^ 감사합니다

**Comment by DonggeunJung** [comment]
시간 아무때나 다 괜찮습니다.

**Comment by bryantson** [comment]
@jparksecurity 님이 두들 만들어서 가능한 시간 고르게 만들었습니다. 여기서 시간 골라주세요. 옆으로 스크롤 하면 더 많은 시간이 나옵니다.

https://doodle.com/poll/w3xhhbt9vskvqcrc?utm_source=poll&utm_medium=link

**Comment by Eunyoungkim0** [comment]
두들에 가능한 시간 표시했습니다!
